Cargo.toml

```diff
@@ -26,7 +26,10 @@ debug = true
 debug = "line-tables-only"
 
 [workspace.dependencies]
-delta_kernel = { version = "0.12.1", features = ["arrow-55", "internal-api"] }
+delta_kernel = { version = "0.12.1", features = [
+    "arrow-55",
+    "internal-api",
+], git = "https://github.com/roeap/delta-kernel-rs", rev = "687e65920e2520524b8a99dc0ebfab971ba7f619" }
```

Comment:
please leave a comment to the pr this corresponds to. won't this also make us unreleasable until next kernel release with it?